web/themes/joinup/prototype/sass/_drupal-hacks.sass

```diff
@@ -182,3 +182,28 @@ form .description
     &:before
       +icon-size(15px)
       content: '\e617'
+
+// Additional styling for dropbutton.
+.dropbutton-action
```

Comment:
It fixes dropdown button on form pages ex. add custom page. It fixes an issue described in ISAICP-3578